apps/worker/src/processors/ai.processor.test.ts

```
@@ -0,0 +1,110 @@
+import type { Job } from 'bullmq'
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)